lib/msal-browser/src/cache/BrowserCacheManager.ts

```diff
@@ -431,8 +434,10 @@ export class BrowserCacheManager extends CacheManager {
      */
     setAccount(account: AccountEntity): void {
         this.logger.trace("BrowserCacheManager.setAccount called");
+        // Remove ID token claims before saving account entity
+        const baseAccount = { ...account, idTokenClaims: undefined };
```

Comment:
Make sure account is cached without ID token claims. This explicit removal is necessary because setAccount can be called outside of the normal browser response handling flow, so this is the only way to make sure the claims aren't cached.

Reply:
Update: Not necessary, went back and made sure setAccount calls are preceded by account entity building that doesn't add ID token claims